workflow/usage.rst

```diff
@@ -439,14 +439,14 @@ Guard Events
 There are a special kind of events called "Guard events". Their event listeners
 are invoked every time a call to ``Workflow::can``, ``Workflow::apply`` or
 ``Workflow::getEnabledTransitions`` is executed. With the guard events you may
-add custom logic to decide what transitions are valid or not. Here is a list
+add custom logic to decide what transitions should be blocked or not. Here is a list
 of the guard event names.
 
 * ``workflow.guard``
 * ``workflow.[workflow name].guard``
 * ``workflow.[workflow name].guard.[transition name]``
 
-See example to make sure no blog post without title is moved to "review"::
+See example to make sure no blog post without title is moved to "reviewed"::
```

Comment:
This sentence is more of a fragment. Something like "This example stops any blog post being transitioned to 'reviewed' if it is missing a title" would be better.

Reply:
Ok np